purbee_backend/backend_source/app.py

```diff
@@ -27,6 +27,116 @@
 app = Flask(__name__)
 
 
+@app.route('/api/community_page/ban', methods=["PUT"])
+def ban_from_community_page():
+    req = request.get_json()
+    data = {"response_message": None}
+    status_code = None
+    if request.method == "PUT":
```

Comment:
I think this if statement is useless because this endpoint only accepts put requests but it will not have a bad effect on the functionality.

Reply:
As you mentioned, it is a design choice.